example-plugins/simple_note_button_plugin/CANVAS_MANIFEST.json

```
@@ -0,0 +1,27 @@
+{
+    "sdk_version": "0.1.4",
+    "plugin_version": "0.1.0",
+    "name": "simple_note_button_plugin",
+    "description": "A simple plugin that adds a hello world button to note headers",
+    "components": {
+        "commands": [],
+        "protocols": [
+            {
+                "class": "simple_note_button_plugin.commands.hello_world_button:HelloWorldButton",
```

We should not name the module (and the directory) "commands", instead let's use "handlers"

```suggestion
                "class": "simple_note_button_plugin.handlers.hello_world_button:HelloWorldButton",
```

---

@copilot please make this change